test/packages/parallel/httpcheck/manifest.yml

```diff
@@ -0,0 +1,76 @@
+format_version: 3.4.0
```

Comment:
This will need to be `3.5.0` as of https://github.com/elastic/package-spec/pull/975.